mkosi

```diff
@@ -1329,6 +1391,18 @@ def prepare_tree(args: CommandLineArguments, root: str, do_run_build_script: boo
             cmdline.write(' '.join(args.kernel_command_line))
             cmdline.write("\n")
 
+        dracut_dir = os.path.join(root, "etc/dracut.conf.d")
+        os.mkdir(dracut_dir, 0o755)
+
+        with open(os.path.join(dracut_dir, "mkosi-hostonly.conf"), "w") as f:
+            f.write("hostonly=no\n")
+
+        with open(os.path.join(dracut_dir, "mkosi-systemd-extras.conf"), "w") as f:
+            f.write(f"install_optional_items+={' '.join(DRACUT_SYSTEMD_EXTRAS)}\n")
+
+        with open(os.path.join(dracut_dir, "mkosi-uefi-stub.conf"), "w") as f:
+            f.write("uefi_stub=/usr/lib/systemd/boot/efi/linuxx64.efi.stub\n")
```

Comment:
I'm not sure this line is necessary, dracut defaults to `/lib/systemd/boot/efi/linuxx64.efi.stub` which should always be appropriately linked.

Reply:
I had one distro that failed without this. I'm not sure which one it was, I'll remove the line and let CI run to figure out which distro so we can only add the file on that distro with a comment so it's clearer why we do this.